src/main/java/uk/gov/hmcts/reform/sscs/util/OtherPartyDataUtil.java

```diff
@@ -43,6 +45,28 @@ public static void assignOtherPartyId(List<CcdValue<OtherParty>> otherParties) {
         }
     }
 
+    public static void processCaseState(SscsCaseData sscsCaseData) {
+        if (sscsCaseData.getIsFqpmRequired() == null
+                || hasDueDateSetAndOtherPartyWithoutHearingOption(sscsCaseData)) {
+            sscsCaseData.setState(State.NOT_LISTABLE);
+        } else {
+            sscsCaseData.setState(State.READY_TO_LIST);
+        }
+    }
+
+    private static boolean hasDueDateSetAndOtherPartyWithoutHearingOption(SscsCaseData sscsCaseData) {
+        return StringUtils.isNotBlank(sscsCaseData.getDwpDueDate())
+                && !everyOtherPartyHasAtLeastOneHearingOption(sscsCaseData);
+    }
+
+    private static boolean everyOtherPartyHasAtLeastOneHearingOption(SscsCaseData sscsCaseData) {
+        if (sscsCaseData.getOtherParties() != null) {
+            return sscsCaseData.getOtherParties().stream().noneMatch(otherParty -> otherParty.getValue().getHearingOptions() == null);
```

Comment:
Is this correct?

Reply:
it is, why you think it isn't?

Reply:
Mike mentioned there would never be a HearingOption object with no hearing options set. If that is the case, we can resolve this comment.